mozecode/dose
# Gm remind

I made the following changes:

- add later npm module to package json and put in demo mode for reminders on every minute as a sample for demo day 
- add better spacing and fix meds appearing in 10 am slot that should not have been, work on reminders in script_list page
- add reminders for prescriptions
- minor change to 10pm reminder

To test:  
- Run ```npm start``` and go to localhost:4000.
- Log in as Santa so that you have medications at all possible times.
- Go to the "See all my prescriptions" view and wait until one of the times listed on the page.  
- You should hear the browser announce what you need to take when and how much.  

STRETCH GOAL ACHIEVED!!!! 🌻
